alert-policies/flutter-web/ClientandServerErrorResponse.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Client and Server Error Response
+
+# Description and details
+description: |+
+  This alert is triggered when the client and server error response is higher than 10 for 5 minutes.
+# Type of alert
+type: STATIC
+
+# NRQL query
+nrql:
+  query: "SELECT uniqueCount(requestUrl) FROM AjaxRequest WHERE httpResponseCode > 399 FACET requestUrl"
```

Comment:
SELECT uniqueCount(requestUrl) FROM AjaxRequest WHERE httpResponseCode > 399

Reply:
Hi @pkosanam,
Updated alert query. kindly review it.